features/fixtures/rails5/app/app/controllers/breadcrumbs_controller.rb

```diff
@@ -0,0 +1,35 @@
+class BreadcrumbsController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  def initialize
+    @cache = ActiveSupport::Cache::MemoryStore.new
+  end
+
+  def handled
+    Bugsnag.notify("Request breadcrumb")
+    render json: {}
+  end
+
+  def sql_breadcrumb
+    User.take
+    Bugsnag.notify("SQL breadcrumb")
+    render json: {}
+  end
+
+  def active_job
+    ApplicationJob.perform_later
+    render json: {}
+  end
+
+  def cache_read
+    @cache.write('test', true)
+    @cache.read('test')
+    Bugsnag.notify("Cache breadcrumb")
+    render json: {}
+  end
+
+  def action_mailer
```

Comment:
It looks like there is no route or maze-runner test that visits this. Did you forget to add something?